app/views/incidents/_form.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= simple_form_for incident do |f| %>
   <%= f.input :title %>
   <%= f.input :attachment, as: :file, label: "Upload an attachment" %>
-  <%= f.input :date %>
+  <%= f.input :date, as: :date, html5: true%>
   <%= f.input :place %>
   <%= f.input :antagonizer %>
```

Comment:
since we do not need antagonizer for todays pitch, it would be a good idea to comment out